src-docs/src/views/health/health.js

```diff
@@ -17,5 +17,9 @@ export default () => (
     <EuiSpacer />
 
     <EuiHealth color="danger">Failure</EuiHealth>
+
+    <EuiSpacer />
+
+    <EuiHealth color="#000000">Dominance</EuiHealth>
```

Comment:
From a docs perspective we like to use the example text to emulate what is happening here. So instead of "Dominance" which is putting a particular meaning to the color, we would just say "Custom color".
```suggestion
    <EuiHealth color="#000000">Custom color as hex</EuiHealth>
```